go.mod

```diff
@@ -133,3 +120,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
Need to split into a types PR

Reply:
Doing that while solving conflicts. 👍🏻 +1